### DISCUSSION TITLE
Home folder may not contain other folders?

### DISCUSSION THREAD

**Discussion by richi** [datenstrom/community#205] [Ask a question] [answered]
Install a vanilla Yellow, put a folder "1.sub" containing a "page.txt" in "1.home" and now try to load "www.yourdomain.xxx/" and a "file not found" error is the result. At least on my apache24, php56 setup. Loading "www.yourdomain.xxx/home" does work flawless.

Looks like the directory named through "ContentHomeDir:" is causing problems as soon as subpages are involved. 

My current workaround is a invisible landing page in "ContentHomeDir:" containing a "Redirect:" to the real "home" which contains subpages...

Any ideas?

Cheers, richi

**Comment by markseuffert** [comment, marked as answer]
Just create subfolders in your content folder, here's an example:

![screen shot 2016-03-12 at 13 35 13](https://cloud.githubusercontent.com/assets/2429165/13722984/5110f3ba-e857-11e5-9d08-f9fecb7cb5fa.png)

Here's a longer explanation: You should be able to create any website structure you want. Because of how the filesystem is organized, the home folder may only contain files. Any other folder may contain both files and folders. Should you make a folder in your home folder, then the home folder will ~~be available under its full name and treated as any other folder~~ return error 500.

See also datenstrom/community#159

**Comment by richi** [comment]
Thanks for the explanation!

The folder mentioned in "ContentHomeDir:" (aka "home" folder) may not contain other folders, only files are allowed, because it has to work as "/" as well (so to speak).

 I guess, this should go into the documentation :)

**Comment by markseuffert** [comment]
Thanks for the suggestion, documentation updated.
